### PR TITLE
Fix non-http prepended urls

### PR DESCRIPTION
**Pull request by alexjoverm**
When either `body.blog || body.html_url` are not prepended with http, github itself does it for you. But when those are retrieved from the url, they still come without the http://, causing the link to be broken.

See this PR from one of my repos for more info: https://github.com/alexjoverm/typescript-library-starter/pull/7

Additionally, I made this PR from windows and was finding a lot of CLRF issues, so I've added a .editorconfig thus that doesn't happen to other people.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#root = true
```
Comment (kentcdodds) [inline .editorconfig]: Please remove this. I prefer to not have unrelated changes in PRs. Feel free to make another PR if you want to add this.

File: lib/contributors/github.js [hunk 2]
```diff
@@ -12,11 +12,14 @@ module.exports = function getUserInfo(username) {
...
     var body = JSON.parse(res.body);
+    var profile = body.blog || body.html_url;
+    profile = profile.indexOf('http') === 0 ? profile : 'http://' + profile;
+
     return {
       login: body.login,
       name: body.name || username,
       avatar_url: body.avatar_url,
-      profile: body.blog || body.html_url
+      profile: profile
```
Comment (kentcdodds) [inline lib/contributors/github.js]: We support Node v4 and above so you can use object shorthand here:

```
{
  // ...
  profile
  // ...
}
```

File: lib/contributors/github.test.js [hunk 3]
```diff
@@ -41,3 +41,19 @@ test('should fill in the name when an empty string is returned', t => {
     t.is(info.name, 'nodisplayname');
   });
 });
+
+test('should append http when no absolute link is provided', t => {
```
Comment (kentcdodds) [inline lib/contributors/github.test.js]: :+1:

**Comment by alexjoverm** [comment]
@kentcdodds I've also realized I could use startsWith instead of indexOf :)

**Comment by kentcdodds** [comment]
Thanks a bunch!

**Comment by jfmengels** [comment]
Thanks a lot @alexjoverm! :tada:

**Pull request merged**